packages/create-svelte/bin.js

```diff
@@ -1,150 +1,3 @@
-					const meta_file = dist(`templates/${dir}/meta.json`);
-					const { title, description } = JSON.parse(fs.readFileSync(meta_file, 'utf8'));
-
-					return {
-						label: title,
-						hint: description,
-						value: dir
-					};
-				})
-			}),
-
-		types: ({ results }) =>
-			p.select({
-				message: 'Add type checking with TypeScript?',
-				initialValue: /** @type {'checkjs' | 'typescript' | null} */ (
-					results.template === 'skeletonlib' ? 'checkjs' : 'typescript'
-				),
-				options: [
-					{
-						label: 'Yes, using TypeScript syntax',
-						value: 'typescript'
-					},
-					{
-						label: 'Yes, using JavaScript with JSDoc comments',
-						value: 'checkjs'
-					},
-					{ label: 'No', value: null }
-				]
-			}),
-
-		features: () =>
-			p.multiselect({
-				message: 'Select additional options (use arrow keys/space bar)',
-				required: false,
-				options: [
-					{
-						value: 'eslint',
-						label: 'Add ESLint for code linting'
-					},
-					{
-						value: 'prettier',
-						label: 'Add Prettier for code formatting'
-					},
-					{
-						value: 'playwright',
-						label: 'Add Playwright for browser testing'
-					},
-					{
-						value: 'vitest',
-						label: 'Add Vitest for unit testing'
-					},
-					{
-						value: 'svelte5',
-						label: 'Try the Svelte 5 preview (unstable!)'
-					}
-				]
-			})
-	},
-	{ onCancel: () => process.exit(1) }
-);
-
-await create(cwd, {
-	name: path.basename(path.resolve(cwd)),
-	template: /** @type {'default' | 'skeleton' | 'skeletonlib'} */ (options.template),
-	types: /** @type {'checkjs' | 'typescript' | null} */ (options.types),
-	prettier: options.features.includes('prettier'),
-	eslint: options.features.includes('eslint'),
-	playwright: options.features.includes('playwright'),
-	vitest: options.features.includes('vitest'),
-	svelte5: options.features.includes('svelte5')
-});
-
-p.outro('Your project is ready!');
-
-if (!options.types && options.template === 'skeletonlib') {
-	const warning = yellow('▲');
-	console.log(
-		`${warning} You chose to not add type checking, but TypeScript will still be installed in order to generate type definitions when building the library\n`
-	);
-}
-
-console.log('Install more integrations with:');
-console.log(bold(cyan('  npx svelte-add')));
-
-console.log('\nNext steps:');
-let i = 1;
-
-const relative = path.relative(process.cwd(), cwd);
-if (relative !== '') {
-	console.log(`  ${i++}: ${bold(cyan(`cd ${relative}`))}`);
-}
-
-console.log(`  ${i++}: ${bold(cyan(`${package_manager} install`))}`);
-// prettier-ignore
-console.log(`  ${i++}: ${bold(cyan('git init && git add -A && git commit -m "Initial commit"'))} (optional)`);
-console.log(`  ${i++}: ${bold(cyan(`${package_manager} run dev -- --open`))}`);
-
-console.log(`\nTo close the dev server, hit ${bold(cyan('Ctrl-C'))}`);
-console.log(`\nStuck? Visit us at ${cyan('https://svelte.dev/chat')}`);
+console.log("'npm create svelte' has been replaced with 'npx sv create'");
```

Comment:
```suggestion
console.warn("'npm create svelte' has been replaced with 'npx sv create'");
```

This makes it more visible in my opinion, though this might be controversial.